src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -149,6 +158,26 @@ export const UserDetailsForm: React.FC<Props> = ({
     )
   }
 
+  const handlePhoneChange = (event: React.FormEvent<HTMLInputElement>) => {
+    const { value } = event.currentTarget
+    const phoneNumber = value
```

Comment:
@mehdielket if I remember correctly a discussion in one of our daily, we have a ticket to add Phone number into `Offer/Checkout` too right?!

Reply:
yes. @mehdielket already finished his part. this PR is for the one currently on prod.

Reply:
👍🏼